samples/src/Universal/Events/AspectEventArgs.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Reflection;
+using Aspects.Universal.Attributes;
+
+namespace Aspects.Universal.Events
+{
+    public class AspectEventArgs : EventArgs
+    {
+        public object Instance { get; set; }
+        public Type Type { get; set; }
+        public MethodBase Method { get; set; }
+        public Func<object[], object> Target { get; set; }

```

Comment:
We have only OnBefore, OnAfter, OnException. **Target** allows to invoke the method directly. I don't understand how this one can be used.